server/apps/immich/src/modules/background-task/background-task.processor.ts

```diff
@@ -1,22 +1,10 @@
+import { assetUtils } from '@app/common/utils';
 import { Process, Processor } from '@nestjs/bull';
-import { InjectRepository } from '@nestjs/typeorm';
-import { Repository } from 'typeorm';
-import { AssetEntity, SmartInfoEntity } from '@app/infra';
 import { Job } from 'bull';
 import { AssetResponseDto } from '../../api-v1/asset/response-dto/asset-response.dto';
-import { assetUtils } from '@app/common/utils';
 
 @Processor('background-task')
```

Comment:
should we change the processor name and the process name to constant in this file as well?

Reply:
Yessir!